extensions/blocks/podcast-player/style.scss

```diff
@@ -119,9 +119,20 @@ $player-float-background: $light-gray-200;
 	}
 
 	.jetpack-podcast-player__audio-player {
+		height: 40px; /* the same as .mejs-container */
 		margin-bottom: $player-grid-spacing;
 	}
 
+	&.is-default .jetpack-podcast-player__audio-player {
+		display: none;
+	}
+
+	.jetpack-podcast-player--audio-player-loading {
+		height: 10px;
+		background: $player-slider-background;
+		margin: 15px $player-grid-spacing;
```

Comment:
You could get rid of the `15px` magic-ish number here by doing:

``` 
.jetpack-podcast-player__audio-player {
	height: 40px; /* the same as .mejs-container */
	margin-bottom: $player-grid-spacing;
 	display: flex;
  	align-items: center; 
}

.jetpack-podcast-player--audio-player-loading {
	height: 10px;
	background: $player-slider-background;
	margin: 0 $player-grid-spacing;
}
```

Reply:
This 15px, while being in the declared in some variable is not where the player gets the size from. The player gets it from the mejs default styles, which are defined outside of this file. The same thing with the 10px height. 

Unless we override mejs styles with those values too, I think it's actually counter intuitive to extract them from here into variables, since they only be controlling the placeholder and not the player itself. Not helping with the consistency.

Reply:
Makes sense 👍